GOVERNANCE.md

```diff
@@ -0,0 +1,360 @@
+based on a [Consensus Seeking Process](#consensus-seeking-process) to add or not those people.
+    * No-one is ever suggested publicly without first discussing the idea with 
+      them first (to make sure that they are okay with it).
+
+
+The PLC has final authority over this project including:
+
+* Project governance and process (including this policy).
+* Contribution policy (shared with the TLC).
+* Budget and legal related issues.
+
+The PLC also should discuss once a month the overall status of the project and any pending or
+upcoming topics.
+
+A PLC member may be removed from the PLC by voluntary resignation, or by a 
+standard PLC motion.
+
+#### Process for Adding PLC Members
+
+1. Invite to the Slack PLC chat room (`selenium-plc`)
+1. Tweet congratulations to the new PLC member from the SeleniumHQ Twitter account
+
+The current PLC members are:
+
+* [@barancev](https://github.com/barancev)
+* [@jimevans](https://github.com/jimevans)
+* [@manoj9788](https://github.com/manoj9788)
+* [@shs96c](https://github.com/shs96c)
+
+
+## Communication Channels
+
+The project maintains various channels for providing information, supporting development
+and enabling communication between team members. Adherence to the project's
+[Code of Conduct](/CODE_OF_CONDUCT.md) is strictly mandatory for all types of communication
+in these channels.
+
+- Twitter Account ([`@seleniumhq`](https://twitter.com/seleniumhq)): for communicating
+and promoting news around the project or project related topics.
+- [Chat Room](https://webchat.freenode.net/#selenium): chat for all Selenium users to seek
+help and support on problems using the project.
+- [Slack](https://seleniumhq.slack.com/join/shared_invite/enQtODAwOTUzOTM5OTEwLTZjZjgzN2ExOTBmZGE0NjkwYzA2Nzc0MjczMGYwYjdiNGQ5YjI0ZjdjYjFhMjVlMjFkZWJmNDYyMmU1OTYyM2Y):
+mirrors the IRC Chat room and is also used by all Selenium users to seek help and support
+on problems using the project.
+- Project Committers Channel, `selenium-committers` in the Slack channel mentioned above:
+private channel for members of the Project Committers team to discuss contributions and
+organise other collaborative efforts.
+- TLC Channel, `selenium-tlc` in the Slack channel mentioned above:
+private channel for TLC members to discuss technical planning and technical project roadmap. 
+- PLC Channel, `selenium-plc` in the Slack channel mentioned above:
+private channel for PLC members to overall project planning, roadmap and related issues. 
+
+## Consensus Seeking Process
+
+The PLC and TLC follow a [Consensus Seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+decision making model.
+
+When an agenda item has appeared to reach a consensus, the moderator will ask
+"Does anyone object?" as a final call for dissent from the consensus.
+
+If an agenda item cannot reach a consensus, a PLC/TLC member can call for either
+a closing vote or a vote to table the issue to the next meeting. The call for a
```

Comment:
Since the word "table" has different meanings in different countries, maybe we should change it to something like "postpone further discussion on the issue until"

Reply:
Changed it as per the suggestion, please have a look and resolve this conversation if things look good for you @titusfortner.